### PR TITLE
Cloud changelog - Nov 10

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: source/install/cloud-changelog.md [hunk 1]
```diff
@@ -3,12 +3,45 @@
...
+ - Added support for navigating through Collapsed Reply Threads via arrow keys.
+ - Added support for focusing the input box in Collapsed Reply Threads while typing.
+ - Added support for blurring the input box in Collapsed Reply Threads on pressing Escape.
+ - Adjusted the channel override desktop notification preference for Threads.
+ - Added a **Click to open thread** setting for all users.
+ - Added lang/tz defaults to Luxon.
+ - User interface is now improved when no text is set for a custom status.
+
+#### Administration
+ - Channel results in channel switcher will include channels from other teams as well as private channels. For users using Bleve or Elasticsearch for autocomplete/search would have to reindex their data to get the new results. Since this can take a long time, it is suggested to disable autocomplete and run indexing in the background. And then when it's complete, re-enable autocomplete.
```
Comment (jasonblais) [inline source/install/cloud-changelog.md:29]: I don't think this applies to Cloud

File: source/install/cloud-changelog.md [hunk 2]
```diff
@@ -3,12 +3,45 @@
...
+
+#### User Interface (UI)
+ - Added online status to profile images on user autocomplete.
+ - App Commands now have an option to open them as modals.
+ - Added support for navigating through Collapsed Reply Threads via arrow keys.
+ - Added support for focusing the input box in Collapsed Reply Threads while typing.
+ - Added support for blurring the input box in Collapsed Reply Threads on pressing Escape.
+ - Adjusted the channel override desktop notification preference for Threads.
+ - Added a **Click to open thread** setting for all users.
+ - Added lang/tz defaults to Luxon.
```
Comment (jasonblais) [inline source/install/cloud-changelog.md:25]: What does this mean?

**Review by jasonblais — CHANGES_REQUESTED** [review]
Two comments, else LGTM, thanks!

File: source/install/cloud-changelog.md (hunk 2 above)
Comment (cwarnermm) [reply source/install/cloud-changelog.md:25]: Is this a reference to supporting Swahili in product translations?

Comment (amyblais) [reply source/install/cloud-changelog.md:25]: @calebroseland may have more details.

File: source/install/cloud-changelog.md [hunk 3]
```diff
@@ -3,12 +3,45 @@
...
+### Bug Fixes
+ - Fixed an issue where API allowed changing the name of the Town Square channel.
+ - Fixed an issue where errors were logged if a user disabled notifications.
+ - Fixed an issue where a channel was not immediately removed from the sidebar when the current user was removed from one.
+
+### Known Issues
+ - Member type is missing from autocomplete [MM-38989](https://mattermost.atlassian.net/browse/MM-38989).
+ - File upload might fail for SVG files [MM-38982](https://mattermost.atlassian.net/browse/MM-38982).
+ - ``CMD+/`` does not close the shortcuts modal [MM-38971](https://mattermost.atlassian.net/browse/MM-38971).
+ - ``Ctrl/Cmd+Shift+A`` shortcut does not open **Account Settings** [MM-38236](https://mattermost.atlassian.net/browse/MM-38236).
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:40]: [This v6.2 terminology update PR](https://github.com/mattermost/mattermost-webapp/pull/9257) includes changes to the CTRL/CMD+Shift+A shortcut from "Account Settings" to "Settings".  If this shortcut should be opening Account Settings, I'll need to update that PR accordingly. CC @esethna for confirmation.

**Review by cwarnermm — COMMENTED** [review]
Thanks! Feedback provided inline.

File: source/install/cloud-changelog.md (hunk 1 above)
Comment (agnivade) [reply source/install/cloud-changelog.md:29]: To confirm, we don't use Elasticsearch in cloud?

**Review by agnivade — APPROVED** [review]

File: source/install/cloud-changelog.md (hunk 2 above)
Comment (calebroseland) [reply source/install/cloud-changelog.md:25]: Sorry, I didn’t think to check country codes before using a short form! Looking at the other release notes—this one could be omitted, as there isn’t a direct change in UX. 

Expanded: “Added language and timezone defaults to Luxon for use in time formatting.”

Comment (amyblais) [reply source/install/cloud-changelog.md]: It seems it's not necessarily needed here, so I've removed it.

File: source/install/cloud-changelog.md (hunk 1 above)
Comment (jasonblais) [reply source/install/cloud-changelog.md]: @agnivade correct, no Elasticsearch in cloud at the mometn

**Review by jasonblais — APPROVED** [review]

File: source/install/cloud-changelog.md (hunk 1 above)
Comment (amyblais) [reply source/install/cloud-changelog.md]: Removing,

**Review by spirosoik — APPROVED** [review]
lgtm

**Review by wiersgallak — APPROVED** [review]

File: source/install/cloud-changelog.md (hunk 3 above)
Comment (cwarnermm) [reply source/install/cloud-changelog.md:36]: @esethna - Should the shortcut key sequence CTRL/CMD+SHIFT+A open the **Settings** dialog or the **Profile** dialog?

**Pull request merged**